compose/cli/command.py

```diff
@@ -107,10 +107,18 @@ def get_config_path(self, file_path=None):
             'fig.yaml',
         ]
 
-        def expand(filename):
-            return os.path.join(self.base_dir, filename)
+        def search_candidates(path):
```

Comment:
Also, I think we should move this to a top-level utility function, and move `supported_filenames` to a top-level constant (`SUPPORTED_FILENAMES`).